fastlane_bot/helpers/txhelpers.py

```diff
@@ -969,13 +974,58 @@ def approve_token_for_arb_contract(self, token_address: str, approval_amount: in
                             base_gas_price=baseFee, max_priority_fee=max_priority, nonce=self.get_nonce()
                         )
                     )
+                    approve_tx = self.sign_transaction(approve_tx)
+                    self.ConfigObj.logger.info(f"Submitting approval for token: {token_address}")
+                    return self.submit_transaction(approve_tx)
                 except Exception as e:
                     self.ConfigObj.logger.info(
                         f"(***2***) Error when building transaction: {e.__class__.__name__} {e}")
             else:
                 return None
-        self.ConfigObj.logger.info(f"Submitting approval for token: {token_address}")
-        return self.submit_transaction(approve_tx)
 
+    async def get_layer_one_gas_fee(self, signed_transaction) -> Decimal:
```

Comment:
The name of a function which is not part of the module's interface (i.e., not designated to be called by other modules) should be prepended with an underscore

Reply:
Added underscore